packages/website/blog/2024-05-27-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -187,11 +187,45 @@ Several rules are changed in significant enough ways to be considered breaking c
   - If you want to have the rule check conditional tests, set its [`ignoreConditionalTests` option](/rules/prefer-nullish-coalescing/#ignoreconditionaltests) to `false` in your ESLint config
 - [feat(eslint-plugin): [no-unused-vars] align catch behavior to ESLint 9](https://github.com/typescript-eslint/typescript-eslint/pull/8971)
   - If you want [`@typescript-eslint/no-unused-vars`](/rules/no-unused-vars) to ignore caught errors, enable its `caughtErrors` option to `'none'` in your ESLint config
+
+#### Replacement of `ban-types`
+
+[`@typescript-eslint/ban-types`](https://typescript-eslint.io/rules/ban-types) has long been one of the more controversial rules in typescript-eslint.
+It served two purposes:
+
+- Allowing users to ban a configurable list of types from being used in type annotations
+- Banning confusing or dangerous built-in types such as `Function` and `Number`
+
+Notably, `ban-types` banned the built-in `{}` ("empty object") type in TypeScript.
+The `{}` type is a common source of confusion for TypeScript developers because it matches _any non-nullable_ object, including primitives like `""`.
```

Comment:
```suggestion
The `{}` type is a common source of confusion for TypeScript developers because it matches _any non-nullable_ value, including primitives like `""`.
```